kubernetes/resource_kubernetes_tokenrequest.go

```diff
@@ -77,16 +77,13 @@ func resourceKubernetesTokenRequestCreateV1(ctx context.Context, d *schema.Resou
 }
 
 func resourceKubernetesTokenRequestReadV1(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-
 	return nil
 }
 
 func resourceKubernetesTokenRequestUpdateV1(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-
-	return resourceKubernetesRoleRead(ctx, d, meta)
+	return resourceKubernetesRoleV1Read(ctx, d, meta)
```

Comment:
This looks wrong, I will open a separate PR to fix.

Reply:
`resourceKubernetesRoleReadV1`

Reply:
Think it should also be for update and read

Reply:
You are right, I did this in a different PR when updated just token request tests. Within this PR I only touched the token request because suddenly it uses the RBAC read function but it should not. In short, it is all fixed in this [PR](https://github.com/hashicorp/terraform-provider-kubernetes/pull/2263).

Good catch!